modules/installation-special-config-storage.adoc

```diff
@@ -46,72 +79,82 @@ Mirroring is available only for user-provisioned infrastructure deployments on {
 ====
 
 [id="installation-special-config-storage-procedure_{context}"]
-== Configuring disk encryption or mirroring
+== Configuring disk encryption and mirroring
 
-.Procedure
+You can enable and configure encryption and mirroring during an {product-title} installation.
 
-To enable boot disk encryption or mirroring during {product-title} deployment:
+.Prerequisites
+
+* You downloaded the {product-title} installation program on your installation node.
+* You installed Butane on your installation node.
++
+[NOTE]
+====
+Machine configs are used to configure the control plane and compute machines. In some situations, such as boot disk mirroring, you can use Butane to create the machine configs for you, making node configuration easier. See the _Creating machine configs with Butane_ section for more information.
```

Comment:
If you changed to For more information, see above, same here.

Reply:
I have now resolved both of these comments.